eosdis_store/stores.py

```diff
@@ -143,6 +147,11 @@ def getitems(self, keys):
         Returns:
             An iterator returning tuples of the input keys to their data or metadata values
         """
+        return dict(self._getitems_generator(keys, **kwargs))
+
+    def _getitems_generator(self, keys, **kwargs):
```

Comment:
I'm not seeing where `**kwargs` are being used here - not sure why the signatures for both `getitems` and `_getitems_generator` are being modified. Most likely I just don't understand Python.

Reply:
`**kwargs` was added to the method signature by the Zarr library, and it was causing our library to fail because it was passing an argument we didn't have declared but is irrelevant to us.  I changed all the signatures to accept kwargs even though we don't use them.